code/modules/recycling/sortingmachinery.dm

```diff
@@ -96,14 +96,12 @@
 		var/atom/A = i
 		A.emp_act(severity)
 
-/obj/item/smallDelivery/attack_self(mob/user as mob)
-	if(wrapped && wrapped.loc) //sometimes items can disappear. For example, bombs. --rastaf0
-		wrapped.loc = user.loc
+/obj/item/smallDelivery/attack_self(mob/user)
+	if(wrapped?.loc == src) //sometimes items can disappear. For example, bombs. --rastaf0
```

Comment:
It shouldn't be possible to wrap an item more than once with the added `is_type_in_list()` check below, but just in case another item or an admin teleports it out, this `loc == src` check will stop it being moved back if the package is unwrapped.